CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## [Unreleased]
 
+### Unfixed
+- Remove auto scroll from add story method
```

Comment:
`Remove auto scroll to bottom when adding a story` would be a better entry here...